### ISSUE TITLE
Error on Arch after latest update

### ISSUE THREAD

**Issue by kgabor545** [xuzhen/dockbarx#6]
Hello!
I get this error and I have no idea how to fix:
```
(dockx:18754): Wnck-CRITICAL **: 20:56:09.343: wnck_set_client_type: changing the client type is not supported.
DockbarX reload
Traceback (most recent call last):
File "/usr/bin/dockx", line 1578, in do_activate
self.window = DockX(application=self, monitor=self.monitor)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/usr/bin/dockx", line 524, in __init__
self.dockbar.load()
File "/usr/lib/python3.11/site-packages/dockbarx/dockbar.py", line 576, in load
self.reload(tell_parent=False)
File "/usr/lib/python3.11/site-packages/dockbarx/dockbar.py", line 644, in reload
self.__add_launcher(identifier, path)
File "/usr/lib/python3.11/site-packages/dockbarx/dockbar.py", line 1555, in __add_launcher
self.__make_groupbutton(identifier=identifier, \
File "/usr/lib/python3.11/site-packages/dockbarx/dockbar.py", line 1006, in __make_groupbutton
group.button.update_state()
File "/usr/lib/python3.11/site-packages/dockbarx/groupbutton.py", line 1516, in update_state
surface = self.icon_factory.surface_update(state_type, force_update=force_update)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/usr/lib/python3.11/site-packages/dockbarx/iconfactory.py", line 168, in surface_update
surface = self.__do_commands(surface, commands)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/usr/lib/python3.11/site-packages/dockbarx/iconfactory.py", line 188, in __do_commands
surface = f(surface, **args)
^^^^^^^^^^^^^^^^^^
File "/usr/lib/python3.11/site-packages/dockbarx/iconfactory.py", line 468, in __command_pixmap
self.temp[name] = f(self.temp[name], **args)
^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/usr/lib/python3.11/site-packages/dockbarx/iconfactory.py", line 421, in __command_if
surface = self.__do_commands(surface, content)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/usr/lib/python3.11/site-packages/dockbarx/iconfactory.py", line 188, in __do_commands
surface = f(surface, **args)
^^^^^^^^^^^^^^^^^^
File "/usr/lib/python3.11/site-packages/dockbarx/iconfactory.py", line 933, in __command_alpha_mask
mask = self.__resize_surface(mask, w, h)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/usr/lib/python3.11/site-packages/dockbarx/iconfactory.py", line 1020, in __resize_surface
im = im.resize((w, h), Image.ANTIALIAS)
^^^^^^^^^^^^^^^
AttributeError: module 'PIL.Image' has no attribute 'ANTIALIAS'
```
And it just hangs here.

Could you help me with this?
Thank you

UPDATE:
I found this thread, with the same issue: https://bbs.archlinux.org/viewtopic.php?id=280016
but the libwinck3 package latest update was on 2022.

**Comment by Archer2011** [comment]
I have the same problem, downgrading python-pillow fixes the issue

**Comment by kgabor545** [comment]
> I have the same problem, downgrading python-pillow fixes the issue

Thank you very much !!!

**Comment by kgabor545** [comment]
The error message still there tho, so it is not even related.

**Comment by xuzhen** [comment]
The Wnck error has been fixed in https://github.com/xuzhen/dockbarx/commit/a8977212355506382d3e8d86f63a2cbfd4cb36d1 a few months ago.

The `AttributeError` is fixed now.  https://github.com/xuzhen/dockbarx/commit/5693b3f3230a3d053158c0b1af05e7623eb44a37

**Comment by Archer2011** [comment]
> 

Thank you very much

**Comment by Archer2011** [comment]
> The Wnck error has been fixed in [a897721](https://github.com/xuzhen/dockbarx/commit/a8977212355506382d3e8d86f63a2cbfd4cb36d1) a few months ago.
> 
> The `AttributeError` is fixed now. [5693b3f](https://github.com/xuzhen/dockbarx/commit/5693b3f3230a3d053158c0b1af05e7623eb44a37)

Thank you very much

**Comment by hrotkogabor** [comment]
It is working great with this fix now.
My question is that, can we release 1.0?
I use dockbarx for many years, so I think it can be released.
